scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -254,32 +225,146 @@ function Install-SecurityDaemon {
     else {
         Write-Host 'Generating config.yaml...'
 
+        if (-not (Test-Path $EdgeDataDirectory)) {
```

Comment:
Are you proposing -Force instead of the Test-Path check? I think Test-Path is cleaner, would trip less privileges issues if any of those were to show up. Besides that, agreed on New-Item vs mkdir.

Reply:
A couple of other places already use `New-Item -Force` instead of `Test-Path + New-Item`, yes. Anyway if you want to keep the `Test-Path` that's fine, but yes change the `mkdir` to `New-Item -Type Directory`

Reply:
Fixed.